site/_includes/footer_custom.html

```diff
@@ -0,0 +1,3 @@
+<p class="text-small text-grey-dk-100 mb-0">
+    Copyright &copy; 2001-2023 IVI Foundation <a href=/privacy_policy.html>privacy policy</a>
```

Comment:
The href value (an attribute value) should be inside double quotes.

Reply:
Also, would we want a bit of formatting?
```suggestion
    Copyright &copy; 2001-2023 IVI Foundation | <a href="/privacy_policy.html">Privacy Policy</a>
```